src/3d/qgsgltf3dutils.cpp

```diff
@@ -495,7 +495,7 @@ static Qt3DCore::QEntity *parseModel( tinygltf::Model &model, const QgsGltf3DUti
       *errors << "Scene contains multiple nodes: only loading the first one!";
   }
 
-  QgsVector3D tileTranslationEcef = QgsGltfUtils::extractTileTranslation( model );
+  const QgsVector3D tileTranslationEcef = rtcCenter + QgsGltfUtils::extractTileTranslation( model );
```

Comment:
I see the rtc center from b3dm vs the translation from GLTF file as mutually exclusive options... but I guess it does not hurt to combine them given that one of them should be always zero :-)